src/backends/sentry_backend_inproc.c

```diff
@@ -114,7 +114,8 @@ shutdown_inproc_backend(sentry_backend_t *UNUSED(backend))
     reset_signal_handlers();
 }
 
-#elif defined SENTRY_PLATFORM_WINDOWS
+#elif defined(SENTRY_PLATFORM_WINDOWS)
```

Comment:
Is this a functional change or just formatting?

Reply:
just formatting